src/probeinterface/probe.py

```diff
@@ -58,7 +69,13 @@ def __init__(self, ndim=2, si_units="um"):
 
         # annotation:  a dict that contains all meta information about
         # the probe (name, manufacturor, date of production, ...)
-        self.annotations = dict(name="")
+        self.annotations = dict()
+        self.annotate(
+            name=name if name is not None else "",
+            serial_number=serial_number if serial_number is not None else "",
+            model_name=model_name if model_name is not None else "",
+            manufacturer=manufacturer if manufacturer is not None else "",
+        )
```

Comment:
I think I would prefer nothing instead of empty string when None.

Reply:
Ok, fair enough ;)